project/driver_controls/src/main.c

```diff
@@ -0,0 +1,116 @@
+#include <stdio.h>
+#include <stdint.h>
+
+#include "adc.h"
+#include "input_interrupt.h"
+#include "driver_state.h"
+#include "soft_timer.h"
+#include "driver_devices.h"
+
+#include "power_state.h"
+#include "pedal_state.h"
+#include "direction_state.h"
+#include "turn_signal_state.h"
+#include "hazard_light_state.h"
+#include "mechanical_brake.h"
+
+#define INPUT_DEVICES 16
```

Comment:
This is defined in `driver_devices.h` as well.

Defines should be prefixed with the module name.